lib/Client.js

```diff
@@ -1987,13 +2003,14 @@ class Client extends EventEmitter {
         if(options.allowedMentions) {
             options.allowed_mentions = this._formatAllowedMentions(options.allowedMentions);
         }
-        if(options.embed) {
-            if(!options.embeds) {
-                options.embeds = [];
-            }
-            options.embeds.push(options.embed);
+        const file = options.file;
+        delete options.file;
+
+        if(options.embed && !options.embeds) {
+            options.embeds = [options.embed];
         }
```

Comment:
this shouldn't be conditional on embeds not being present, they should be combined as the code was before